sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubMetricsProvider.cs

```diff
@@ -162,9 +162,9 @@ private static long GetUnprocessedEventCount(PartitionProperties partitionInfo,
             }
 
             // Legacy checkpoint support
-            if (checkpoint != null && checkpoint.Offset == null && partitionInfo.LastEnqueuedSequenceNumber >= 0)
+            if (checkpoint != null && checkpoint.Offset == null && partitionInfo.LastEnqueuedSequenceNumber == 0)
```

Comment:
With this change, do we need additional tests?  Since the change to the SDK with the format change did not cause any test failures, it seems we're not validating some assumptions.